File: twiml/voice/sip/sip-8/sip-8.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial();
        dial
            .Sip(new Uri("sip:kate@example.com?x-mycustomheader=foo&x-myotherheader=bar"));
        response.Append(dial);

        Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/sip/sip-8/sip-8.5.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\TwiML\VoiceResponse;

$response = new VoiceResponse();
$dial = $response->dial('');
$dial->sip('sip:kate@example.com?mycustomheader=foo&myotherheader=bar');

echo $response;
